build.gradle.kts

```diff
@@ -24,6 +24,10 @@ kotlin {
             dependencies {
                 implementation("com.squareup.okio:okio:3.9.1")
                 implementation("net.thauvin.erik.urlencoder:urlencoder-lib:1.6.0")
+                // Overridig coroutines' version to solve a problem with WASM JS tests.
+                // See https://kotlinlang.slack.com/archives/CDFP59223/p1736191408326039?thread_ts=1734964013.996149&cid=CDFP59223
+                // TODO: remove this workaround in https://github.com/krzema12/snakeyaml-engine-kmp/issues/337
+                runtimeOnly("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.10.1")
```

Comment:
issue: we should add this dependency to the `commonTest` source set because we don't use them in the main library

Reply:
Oops, right! Moved.